src/Static/Db.elm-template

```diff
@@ -1,11 +1,12 @@
 module Static.Db exposing (Db, db)
 
-import Data.Food.Db as FoodDb
+import Data.Food.Explorer.Db as ExplorerDb
+import Data.Food.Builder.Db as BuilderDb
 import Data.Textile.Db as TextileDb
 
 
 type alias Db =
-    { foodDb : FoodDb.Db, textileDb : TextileDb.Db }
+    { builderDb : BuilderDb.Db, explorerDb : ExplorerDb.Db,  textileDb : TextileDb.Db }
```

Comment:
Hmm that means this file contains 3 times the impact definitions ; I wonder how we could have it parsed/loaded and defined only once

Reply:
We have the impacts in each "db" for two reason I believe: 
- in case the other databases aren't loaded (eg if at some point we'd rather not load the textile DB on the recipe builder page)
- to have access to it without passing around the local db and the "textile db" (or impacts db)

It would probably make sense to refactor that at a later point, indeed, but we might want to think a bit about how to architecture it, and i'd rather do it in a future PR.